src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
```

Comment:
It's a bad practice to use global variables if you don't need them. You are using `body` only in the `successHandler` and `errorHandler` functions. So it's better to move this variable into these functions.